specification/resources/resource-manager/Microsoft.Authorization/stable/2016-09-01/locks.json

```diff
@@ -812,6 +812,46 @@
         },
         "x-ms-odata": "#/definitions/ManagementLockObject"
       }
+    },
+    "/{scope}/providers/Microsoft.Authorization/locks": {
```

Comment:
This swagger already has definitions for list at subscription level, resource group level and extension resource level. Please could you respond to clarify what sort of string you expect {scope} to be?

Reply:
For consistency, as all the put/get/delete methods provide subscription, resource group, resources, scope four levels. However, when listing, the scope one is missing.

From user's perspective, all the four levels results in the same URI, I don’t need to care whether the id is resources, resource group or subscription when I use the scope one.